dll/win32/shell32/utils.cpp

```diff
@@ -504,3 +504,29 @@ SHOpenPropSheetA(
     return SHOpenPropSheetW(pszCaptionW, ahKeys, cKeys, pclsidDefault,
                             pDataObject, pShellBrowser, pszStartPageW);
 }
+
+/*************************************************************************
+ *  Activate_RunDLL [SHELL32.105]
+ *
+ * Every parameter is unused.
```

Comment:
Can you at least tell what this function seems to be used for?

Reply:
Fixed.

```c
/*************************************************************************
 *  Activate_RunDLL [SHELL32.105]
 *
 * Unlocks the foreground window and allows the shell window to become the
 * foreground window. Every parameter is unused.
 */
```